src/lib/utils/common/getWeekNumberByMonth.ts

```diff
@@ -0,0 +1,69 @@
+function getWeekNumberByMonth(dateFormat: Date) {
+  const inputDate = new Date(dateFormat);
+
+  // 인풋의 년, 월
+  let year = inputDate.getFullYear();
+  let month = inputDate.getMonth() + 1;
+
+  // 목요일 기준 주차 구하기
+  const weekNumberByThurFnc = (paramDate: Date) => {
+    const year = paramDate.getFullYear();
+    const month = paramDate.getMonth();
+    const date = paramDate.getDate();
+
+    // 인풋한 달의 첫 날과 마지막 날의 요일
+    const firstDate = new Date(year, month, 1);
+    const lastDate = new Date(year, month + 1, 0);
+    const firstDayOfWeek = firstDate.getDay() === 0 ? 7 : firstDate.getDay();
+    const lastDayOfweek = lastDate.getDay();
+
+    // 인풋한 달의 마지막 일
+    const lastDay = lastDate.getDate();
+
+    // 첫 날의 요일이 금, 토, 일요일 이라면 true
+    const firstWeekCheck =
+      firstDayOfWeek === 5 || firstDayOfWeek === 6 || firstDayOfWeek === 7;
+    // 마지막 날의 요일이 월, 화, 수라면 true
+    const lastWeekCheck =
+      lastDayOfweek === 1 || lastDayOfweek === 2 || lastDayOfweek === 3;
+
+    // 해당 달이 총 몇주까지 있는지
+    const lastWeekNo = Math.ceil((firstDayOfWeek - 1 + lastDay) / 7);
```

Comment:
```suggestion
    const NumberOflastWeek = Math.ceil((firstDayOfWeek - 1 + lastDay) / 7);
```
number와 no 변수명을 혼용하여 가독성이 떨어지는것 같습니다.

Reply:
둘중 하나로 통일하시면 좋을것 같습니다.